src/ui/QuerySummary/QuerySummary.ts

```diff
@@ -234,45 +234,20 @@ export class QuerySummary extends Component {
     }
   }
 
-  private updateNoResultsContainerSnapshot() {
+  private updateNoResultsSnapshot() {
```

Comment:
Since it's now called only once in the constructor, I think `update` is not the correct word. `setup` or `configure` would be better.